### DISCUSSION TITLE
Issue-labels: More? Can we help?

### DISCUSSION THREAD

**Discussion by Jonta** [AnySoftKeyboard/AnySoftKeyboard#2789] [Q&A]
- 528 [open issues](https://github.com/AnySoftKeyboard/AnySoftKeyboard/labels) atm
- Only [4+8 (may overlap?) with labels](https://github.com/AnySoftKeyboard/AnySoftKeyboard/labels)
- 11 labels currently available, not including milestones

528 seems like an _overwhelming_ amount to keep track of without using labels or a different sort of overview

I'd be happy to label a bunch of existing ones, but don't currently have the permissions to do so, even on the issues I've reported myself

Issues reported by me (as "credentials"): https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues?q=is%3Aissue+author%3AJonta

=)

**Comment by Jonta** [comment]
@menny Poke. =)

**Reply by menny** [reply]
thanks for the offer.
I've sent you an invite to the Developers team, which will allow you to triage issues and such.

**Reply by Jonta** [reply]
Thank you

**Comment by Jonta** [comment]
Follow up posted here: https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/3023
